feat: document plugin type badges (#14)
camunda/camunda-modeler-plugin-example: README.md

## Before you Publish

> * [ ] Clearly state which Camunda Modeler version your plug-in is compatible with and which type of plugin it is by changing the badges at the top of this README
>   * Mark your plugin using a type badge, so to show your users, what Camunda Modeler features it extends. You can find a best-practice list of typical badges [below](#badges-to-indicate-typical-plugin-types).
>   * If your plugin is `BPMN`/`DMN`/`React` but also includes menu entries or styles, just use `BPMN`/`DMN`/`React` in the badge.

Reviewer: Let's simplify

```suggestion
* [ ] Clearly state which Camunda Modeler version your plug-in is compatible with and which type of plugin it is by changing the badges at the top of this README. See [best-practice examples below](#badges-to-indicate-typical-plugin-types).
```
